src/common/logging/Logging.h

```diff
@@ -16,59 +17,24 @@
 #define ARRAY_SIZE(a) (sizeof(a) / sizeof(a[0]))
 #endif
 
-typedef void* OSCONFIG_LOG_HANDLE;
+typedef struct OsConfigLog OsConfigLog;
```

Comment:
Please do not rename this structure. As commented separately, we do not want callers to get the structure, we want the callers to get an opaque handle. 

Let's discuss separately the pros and cons of changing from opaque handle to well defined type. I want to incorporate this as part of my work for the logging improvements that is in progress right now.